Orange/widgets/visualize/owscatterplot.py

```diff
@@ -179,22 +182,36 @@ def __init__(self):
         self.sampling.setVisible(False)
 
         box = gui.vBox(self.controlArea, "Points")
+        color_model = DomainModel(
+            order=("(Same color)", dmod.Separator) + dmod.SEPARATED,
+            valid_types=dmod.PRIMITIVE)
         self.cb_attr_color = gui.comboBox(
             box, self, "graph.attr_color", label="Color:",
             emptyString="(Same color)", callback=self.update_colors,
```

Comment:
When is the first value "just a string"? This is DomainModel — It's about Variables of the Domain. For arbitrary contents, PyListModel applies. Why would we not want a uniform UX throughout Orange? Why would you put the default option last? Why wouldn't you separate the default from the rest?

You hate this repetition too? Invent an argument for it.